taxonomies/categories.txt

```diff
@@ -32341,7 +32350,11 @@ fr:Yaourts au Bifidus
 nl:Bifidus yoghurts
 
 <en:Bifidus yogurts
-fr:Yaourts au bifidus nature
+fr:Yaourts au bifidus nature, Lait fermenté de type yaourt nature au bifidus, spécialité laitière de type yaourt nature au bifidus
+en:Sweetened plain bifidus yogurts, Yogurt type plain fermented milk with sugar with bifidus, Yogurt type plain dairy specialty with sugar with bifidus
```

Comment:
We always put the en: entry first, to make it the canonical name of the category.

Reply:
Roger that, I will fix it in the next commit